config/config.yml

```diff
@@ -172,6 +172,7 @@ RATE_LIMIT_PER_NGINX_UPSTREAM:
   unicorn_elastic_bookmarks:
     limit: 300
     period: 300
+RATE_LIMIT_SAFELIST: [ '127.0.0.0/8', '10.0.0.0/8' ]
```

Comment:
Could you use double quotes on the strings here, please?

Reply:
Can you also remove the space after `[` and the one before `]`?